### PR TITLE
Added perl-tk dep

### PR DESCRIPTION
**Pull request by mtekman**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [/] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Here until conda-forge includes `perl-tk` themselves, at which point this should be deleted.

### CODE REVIEW

File: recipes/perl-tk/804.034/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+   #  - Tk::Text::Tag
+   #  - Tk::TextEdit
+   #  - Tk::TextList
+   #  - Tk::TextUndo
+   #  - Tk::Tiler
+   #  - Tk::TixGrid
+   #  - Tk::Toplevel
+   # - Tk::Trace
+    # - Tk::Tree
+    # - Tk::Widget
```
Comment (bgruening) [inline recipes/perl-tk/804.034/meta.yaml]: Can you now, include these tests again? Or should they be removed?

Comment (mtekman) [reply recipes/perl-tk/804.034/meta.yaml]: okay, will try

**Comment by mtekman** [comment]
Unable to debug this. Tests fail, but I cannot see where or why

**Comment by mtekman** [comment]
Looking at the build and meta files in [this](https://anaconda.org/Eumetsat/perl-tk/files) anaconda package leads me to believe that the testing does not need to be thorough.

**Comment by bgruening** [comment]
the error was:
```
Can't load '/opt/conda/conda-bld/perl-tk_1508006604813/_t_env/lib/perl5/site_perl/5.22.0/x86_64-linux-thread-multi/auto/Tk/PNG/PNG.so' for module Tk::PNG: libpng16.so.16: cannot open shared object file: No such file or directory at /opt/conda/conda-bld/perl-tk_1508006604813/_t_env/lib/perl5/5.22.0/x86_64-linux-thread-multi/DynaLoader.pm line 193.
```

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**